doc/compiler.md

```diff
@@ -7,25 +7,26 @@ The ProtoDef compiler can convert your protocol JSON into javascript code that c
 Let's take a simple ProtoDef definition and convert it to use the ProtoDef compiler:
 
 ProtoDef:
-```javascript
-const ProtoDef = require('protodef').ProtoDef
+```js
+
+import { ProtoDef } from 'protodef'
 
 // Create a ProtoDef instance
 const proto = new ProtoDef()
-proto.addTypes(require('./protocol.json'))
+proto.addTypes(import('./protocol.json'))
```

Comment:
I think dynamic imports return a promise that wraps the imported value. So this should not work. You would have to refractor the entire file to use async and then use await.

Reply:
Yep that an issue

Reply:
this file can be JSON.parse'd from fs.readFileSync

Reply:
Also I think FS is the better option